tests/test_prompts.py

```diff
@@ -102,7 +102,7 @@ def test_generic_math_problem_augmentation_prompt():
 After the problem is completed finish your response right away.<|eot_id|><|start_header_id|>assistant<|end_header_id|>
 
 """
-    assert prompt.fill({"problem": "What's the meaning of life?"}) == expected_prompt
+    assert prompt.fill({"problem": "What's the meaning of life?"}, format_as_string=True) == expected_prompt
```

Comment:
please check the documentation for instances of prompt.fill and also prepare_data utils (that's used for sft), they all need to be updated with this

Reply:
Yup, I fixed all the tests. Let me fix the docs.

Reply:
Should be good to merge if the tests pass!